pkg/blueprint/installer_customizations.go

```diff
@@ -0,0 +1,6 @@
+package blueprint
+
+type InstallerCustomization struct {
+	Unattended        bool `json:"unattended,omitempty" toml:"unattended,omitempty"`
+	WheelSudoNopasswd bool `json:"wheel-sudo-nopasswd,omitempty" toml:"wheel-sudo-nopasswd,omitempty"`
```

Comment:
That's one mighty variable name :) I hate to suggest it, but maybe sudo needs its own section? I'm betting on there being other common sudo options -- personally I always enable access for wheel, but always require the password.

Reply:
A sudo section might be a good idea.  It's a bit strange though because we're making our own `%post` snippet, so we'll have to be a bit careful with what we put in there, but at this point I think it makes sense.